aea/mail/common.py

```diff
@@ -0,0 +1,160 @@
+# -*- coding: utf-8 -*-
```

Comment:
just moved from the above? Or there are changes?

Reply:
reverted